apps/newsletters-api/src/app/headers.ts

```diff
@@ -0,0 +1,49 @@
+import type { FastifyReply, FastifyRequest } from 'fastify';
+import { isServingUI } from '../apiDeploymentSettings';
+
+type TtlSettings = {
+	cacheMaxAge: number;
+	surrogateMaxAge: number;
+};
+
+const newsletterTtl: TtlSettings = {
+	cacheMaxAge: 60,
+	surrogateMaxAge: 360,
```

Comment:
In the PR description you imply this is 5 minutes, but 360s is 6 minutes. Is there any need for the cache time to be this long, would 5s not be sufficient?

Reply:
I'm not sure what the ideal time would be but 5 minutes probably is too long. I remember that there was some concern that the newsletters api is effectively public. I'd be a bit wary of going from the default TTL of 1 hour to 5s, but I don't know if I'm being over cautious on that.

Looking at it from the other direction - I don't think there are any uses of the newsletters API that would reasonably require the response to be less than 1 minute old, so maybe 60s is a good cache time?